libs/providers/flagd/src/lib/service/grpc/service.ts

```diff
@@ -126,4 +130,9 @@ export class GRPCService implements Service {
       };
     }
   }
+
+  private convertContext(context: EvaluationContext): Struct {
+    // JsonValue closely matches EvaluationContext, this is a safe cast.
+    return Struct.fromJson(context as JsonValue);
+  } 
```

Comment:
Reduced some duped code here, also instead of stringifying, then converting to struct, we just convert directly to struct which should improve performance.